pyquil/tests/test_operator_estimation.py

```diff
@@ -599,3 +599,54 @@ def test_ratio_variance_array():
     var_b = np.array([0.05, 0.5, 5.0])
     ab_ratio_var = ratio_variance(a, var_a, b, var_b)
     np.testing.assert_allclose(ab_ratio_var, np.array([0.028125, 0.0028125, 0.00028125]))
+
+
+def test_measure_observables_noisy_asymmetric_readout_plus_one():
+    qc = get_qc('9q-qvm')
+    expt1 = ExperimentSetting(TensorProductState(plusX(0)), sX(0))
+    expt2 = ExperimentSetting(TensorProductState(plusY(0)), sY(0))
+    expt3 = ExperimentSetting(TensorProductState(plusZ(0)), sZ(0))
+    p = Program()
+    p.define_noisy_readout(0, p00=0.99, p11=0.80)
+    qubs = [0]
+    tomo_expt = TomographyExperiment(settings=[expt1, expt2, expt3], program=p, qubits=qubs)
+
+    for res in measure_observables(qc, tomo_expt, n_shots=1000,
+                                   readout_symmetrize='exhaustive',
+                                   calibrate_readout='plus-eig'):
+        assert np.isclose(1.0, res.expectation, atol=1.e-1)
```

Comment:
This may be a bit pedantic but `0.1` or `1.0e-1` or `1e-1` is preferable to `1.e-1`, with `0.1` or `1e-1` being better choices, style-wise (IMHO).

I also think `0.1` is too generous in magnitude.

Reply:
Instead of using `np.isclose` for a single sample, maybe we can try running many times and looking at the average? It is probably a more meaningful test.  I am not sure what the run time is for this, but I won't imagine running 100 samples of 1k shots is that expensive a test.

Reply:
It seems like having 50 runs of 1000 shots each should be enough to have an `atol` of 0.01 for the average of those runs and have the test pass almost always (from the perspective of statistical fluctuations when everything is working well). That only takes 10s to run on a QMI.

Reply:
For some reason, it's taking closer to 30s for me (50 runs of 1000 shots each). 100 runs brings me closer to `1e-2`, but it takes ~1 min. I'll leave it at 50 runs, 1000 shots each, with an atol of `2e-2`.